src/test/java/bio/terra/service/snapshot/flight/create/NotifyUserOfSnapshotCreationStepTest.java

```diff
@@ -25,18 +27,21 @@ class NotifyUserOfSnapshotCreationStepTest {
   @Mock private SnapshotBuilderService snapshotBuilderService;
   @Mock private SnapshotRequestDao snapshotRequestDao;
   @Mock private IamService iamService;
+  private static final String TOKEN = "some-access-token";
+  private static final AuthenticatedUserRequest TEST_USER =
+      AuthenticationFixtures.userRequest(TOKEN);
```

Comment:
Unless we need a specific token, the usual approach is to use `randomUserRequest()`
```suggestion
  private static final AuthenticatedUserRequest TEST_USER =
      AuthenticationFixtures.randomUserRequest();
```

The token can always be retrieved later by using `TEST_USER.getToken()`